modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase

```diff
@@ -66,12 +66,70 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6087. RemoveAChannelFilter"
-	@ignore = "true"
 	@priority = "3"
 	test RemoveAChannelFilter {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6087 RemoveAChannelFilter
+		CommerceNavigator.gotoPortlet(
+			category = "Store Management",
+			portlet = "Channels");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewChannelsEntry(
+			channelCurrency = "US Dollar",
+			channelName = "Test Channel",
+			channelType = "Site",
+			siteName = "Guest");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Store Management",
+			portlet = "Channels");
+
+		Refresh();
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Visibility");
+
+		Click(
+			key_toggleSwitch = "ojma___toggle-switch",
+			locator1 = "ToggleSwitch#ENABLE_SPECIFIC_TOGGLE_SWITCH");
```

Comment:
I have requested changes to the locator in this PR: https://github.com/liferay-commerce/liferay-portal/pull/903